exla/lib/exla.ex

```diff
@@ -6,11 +6,15 @@ defmodule EXLA do
 
   ## XLA binaries
 
-  EXLA relies on the [XLA](https://github.com/elixir-nx/xla) package to
-  provide the necessary XLA binaries. Whenever possible it tries to download
-  precompiled builds, but you may need to build from source if there is no
-  version matching your target environment. For more details, including
-  GPU/TPU support see [the usage section](https://github.com/elixir-nx/xla#usage).
+  EXLA relies on the `XLA` package to provide the necessary XLA binaries.
```

Comment:
This is going to link to the module itself, which is not what we want. Should we explicitly link to https://hexdocs.pm/xla instead?

Reply:
Ignore me, I saw the other PR.

Reply:
We actually do want to link to the module, so that the URL has the relevant XLA version. With https://github.com/elixir-nx/xla/pull/103 the XLA module docs include all the relevant information. WDYT?